SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialActorChannel.cpp

```diff
@@ -1229,6 +1227,18 @@ void USpatialActorChannel::OnCreateEntityResponse(const Worker_CreateEntityRespo
 			"Actor %s, request id: %d, entity id: %lld, message: %s"), *Actor->GetName(), Op.request_id, Op.entity_id, UTF8_TO_TCHAR(Op.message));
 		break;
 	}
+
+	if (static_cast<Worker_StatusCode>(Op.status_code) == WORKER_STATUS_CODE_SUCCESS &&
+		GetDefault<USpatialGDKSettings>()->bEnableUserSpaceLoadBalancing &&
+		Actor->IsA<APlayerController>())
+	{
+		// With USLB, we want the client worker that results in the spawning of a PlayerController to claim the
+		// PlayerController entity as a partition entity so the client can become authoritative over necessary
+		// components (such as client RPC endpoints, heartbeat component, etc).
+		const Worker_EntityId ClientSystemEntityId = SpatialGDK::GetConnectionOwningClientSystemEntityId(Cast<APlayerController>(Actor));
+		check(ClientSystemEntityId != SpatialConstants::INVALID_ENTITY_ID);
+		Sender->SendClaimPartitionRequest(ClientSystemEntityId, Op.entity_id);
```

Comment:
Curious - can Claim Partition Requests be made from any worker?

Reply:
Any worker with permissions to send system entity-commands, so currently the client can too because we set (idk why tho):
```
    {
      "worker_type": "UnrealClient",
      "permissions": [
        {
          "all": {}
        }
      ]
    }
```